nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py

```diff
@@ -384,6 +384,7 @@ def model_provider_func(self, pre_process, post_process):
                 use_flash_attention=self.cfg.get('use_flash_attention', False),
                 megatron_legacy=self.cfg.get('megatron_legacy', False),
                 seq_len_interpolation_factor=self.cfg.get('seq_len_interpolation_factor', None),
+                base=self.cfg.get('base', 10000),
```

Comment:
Should we change the name to `rotary_base` (similar to `rotary_percentage`), so people can know what is this param used for?

Reply:
Done